catalog/ui/search-ui/standard/server.js

```diff
@@ -20,18 +20,12 @@ var app = express();
 app.use(require('connect-livereload')());
 
 // our compiled css gets moved to /target/webapp/css so use it there
-app.use('/css',express.static(__dirname + '/target/webapp/css'));
-app.use('/lib',express.static(__dirname + '/target/webapp/lib'));
-app.use(express.static(__dirname + '/src/main/webapp'));
-app.use(express.static(__dirname + '/target/webapp/'));
+app.use('/search/standard/', express.static(__dirname + '/target/webapp/'));
+app.use('/search/standard/', express.static(__dirname + '/src/main/webapp'));
 
```

Comment:
You'll need to revert all this stuff back when you make the change I recommended above.